adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -212,16 +213,25 @@ private void validateAcquireTokenRequest(final AuthenticationRequest authenticat
      * Perform authority validation.
      * True if the passed in authority is valid, false otherwise.
      */
-    private void validateAuthority(final URL authorityUrl, final UUID correlationId) throws AuthenticationException {
+    private void validateAuthority(final URL authorityUrl, @Nullable final String domain, final UUID correlationId) throws AuthenticationException {
         if (mAuthContext.getIsAuthorityValidated()) {
             return;
         }
 
         Logger.v(TAG, "Start validating authority");
         mDiscovery.setCorrelationId(correlationId);
-        mDiscovery.validateAuthority(authorityUrl);
 
-        Logger.v(TAG, "The passe in authority is valid.");
+        Discovery.verifyAuthorityValidInstance(authorityUrl);
+
+        // if the domain is null, the AuthenticationRequest being performed is using a GUID
+        // and AD FS validation cannot be performed
+        if (UrlExtensions.isADFSAuthority(authorityUrl) && domain != null) {
```

Comment:
We can simply check if it's adfs authority and if the request is silent to skip authority validation. Also, i would prefer a log message saying that we're skipping the validation.